tests/driver_soft_spi/soft_spi_params.h

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
This is thought to be the way to let the use/developer set the parameters for the soft SPI.